config/local.yaml

```diff
@@ -282,7 +282,7 @@ checks:
     type: settlements
   ci-smoke:
     options:
-      content-size: 50000
+      file-sizes: [3072, 30720, 307200, 31457280, 104857600, 314572800] # [3KB, 30KB, 300KB, 30MB, 100MB, 300MB]
```

Comment:
For the local.yaml, keep some smaller sizes, maybe max to 3 mb. 
Local deployments do not have persistant-storage supported.